en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/S/Sequester.md

```diff
@@ -9,8 +26,8 @@ _7th-level transmutation_
 **Duration:** Until dispelled
 
 You magically hide away a willing creature or object.
-The target becomes invisible, and it cannot be traced or detected by divination or [Scrying](#Scrying_scrying) sensors.
+The target becomes [invisible](#Conditions_invisible), and it cannot be traced or detected by divination or [Scrying](#Scrying_scrying) sensors.
 If the target is a living creature, it falls into a state of suspended animation and stops aging.
 
-The spell ends when the target takes damage or a condition you set occurs.
-The condition must pertain to creatures or objects within 1 mile of the target or be visible within that distance.
+The spell ends when the target takes damage or when a condition you set occurs.
+The condition must pertain to creatures or objects within 1 mile of the target or be observable within that distance.
```

Comment:
I made this change to better align with the _pinpoint_ mechanic; this does however change the spell mechanically, since e.g. you can now also determine an audible or olfactory trigger. It's _technically_ more powerful than either the WotC or the A5e version, but I don't think it's gamebreaking.

Does this make sense to you? And do you agree with my assertion?